src/Monolog/Handler/FilterHandler.php

```diff
@@ -49,6 +49,8 @@ class FilterHandler extends Handler implements ProcessableHandlerInterface, Rese
     protected $bubble;
 
     /**
+     * @psalm-param callable(array, HandlerInterface): HandlerInterface|HandlerInterface $handler
```

Comment:
should be array|null, as per the comment at line 54

Reply:
Fixed.